meta-ostro/recipes-image/images/ostro-image-swupd.bb

```diff
@@ -105,7 +115,8 @@ SWUPD_IMAGES ?= " \
 # In practice the same as "all" at the moment, but conceptually different
 # and thus defined separately.
```

Comment:
Please remove the comment. With your change, the "dev" image no longer has the same content as the "all" image.